app/src/main/java/org/jellyfin/mobile/webapp/RemotePlayerService.kt

```diff
@@ -340,7 +346,18 @@ class RemotePlayerService : Service(), CoroutineScope {
             }.build()
 
             // Post notification
-            notificationManager.notify(MEDIA_PLAYER_NOTIFICATION_ID, notification)
+            if (AndroidVersion.isAtLeastQ) {
+                startForeground(
+                    MEDIA_PLAYER_NOTIFICATION_ID,
+                    notification,
+                    ServiceInfo.FOREGROUND_SERVICE_TYPE_MANIFEST
+                )
+            } else {
+                startForeground(
+                    MEDIA_PLAYER_NOTIFICATION_ID,
+                    notification
+                )
+            }
```

Comment:
Could use [`ServiceCompat.startForeground()`](https://developer.android.com/reference/androidx/core/app/ServiceCompat#startForeground(android.app.Service,int,android.app.Notification,int)) instead.

Reply:
True, however the [`ServiceInfo.FOREGROUND_SERVICE_TYPE_MANIFEST`](https://developer.android.com/reference/android/content/pm/ServiceInfo#FOREGROUND_SERVICE_TYPE_MANIFEST) constant was only added in API level 29, so it either requires a version check, or has to be duplicated locally. I think I prefer the current way over that. What's your preference?

Reply:
Huh, I'd have expected a compat version of that to exist. Since it doesn't, I agree and will merge it as is.